src/screens/ContactScreen/index.js

```diff
@@ -195,9 +252,14 @@ const ContactScreen = ({navigation}) => {
     <SafeAreaView style={styles.container}>
       <StatusBar translucent={false} />
       <Header
-        title={StringConstant.chatTabHeaderCreateChatButtonText}
+        title={
+          isAnon
+            ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+            : StringConstant.chatTabHeaderCreateChatButtonText
+        }
         containerStyle={styles.containerStyle}
         subTitle={'Next'}
+        subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(isAnon)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The title of the `Header` component is now conditionally rendered based on the `isAnon` state variable. This is a good practice as it allows the component to display different titles based on the context it is used in.
```diff
+ title={
+   isAnon
+     ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+     : StringConstant.chatTabHeaderCreateChatButtonText
+ }
+ containerStyle={styles.containerStyle}
+ subTitle={'Next'}
+ subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(isAnon)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->